bin/enrich/json.js

```diff
@@ -171,10 +179,18 @@ const writeInFileJSON = async (tab) => {
  * starts the enrichment process for files JSON
  * @param {*} readStream read the stream of the file you want to enrich
  */
-const enrichmentFileJSON = async (outFile, readStream) => {
+const enrichmentFileJSON = async (outFile, readStream, args) => {
+  if (args.attributes) {
+    checkAttributesJSON(args.attributes);
+  }
+  createFetchAttributes();
   out = outFile;
   let loaded = 0;
-
+  // empty the file
+  const ifFileExist = await fs.pathExists(out);
```

Comment:
`ifFileExist` is not a variable that contains a condition, should be `fileExists`.